python/dgl/graphbolt/impl/ondisk_dataset.py

```diff
@@ -13,7 +13,16 @@
     TorchBasedFeatureStore,
 )
 
-__all__ = ["OnDiskDataset"]
+__all__ = ["OnDiskDataset", "preprocess_ondisk_dataset"]
+
+
+def preprocess_ondisk_dataset(metadata_path: str) -> str:
```

Comment:
make it internal or external?

Reply:
Considering some advanced user wants to preprocess explicitly in advance, make it external?